src/JoomlaBrowser.php

```diff
@@ -897,4 +911,40 @@ public function disableStatistics()
         $I->waitForElement(['link' => 'Never'], 60);
         $I->click(['link' => 'Never']);
     }
+
+    /**
+     * Function to change the status of individual object
+     *
+     * @param  object   $selector   Object selected
+     * @param  Mixed    $status     Status to be selected
+     *
+     * @return void
+     */
+    public function changeIndividualStatus($selector, $status) {
+        $I = $this;
+        $I->debug('Try to select ' . $status . ' from the table');
+        $I->click(['xpath' => "//table[@class='table table-striped']//tbody//tr//td[@class='nowrap has-context']//a[contains(text(), '$selector')]"]);
```

Comment:
Is //td[@class='nowrap has-context']//a[contains(text(), '$selector')] not enough? Any reason to include table class? :) The whole xpath seems too complicated